libs/wingsdk/src/std/resource.ts

```diff
@@ -224,7 +225,7 @@ export abstract class Resource extends Construct implements IResource {
     }
 
     throw new Error(
-      `unable to serialize immutable data object of type ${obj.constructor?.name}`
+      `objects of type '${obj.constructor?.name}' cannot be referenced from an inflight scope`
```

Comment:
In case it's not an object
```suggestion
      `objects of type '${obj.constructor?.name ?? typeof obj}' cannot be referenced from an inflight scope`
```

Reply:
👍